components/dashboard/src/workspaces/Workspaces.tsx

```diff
@@ -32,25 +32,43 @@ export class Workspaces extends React.Component<WorkspacesProps, WorkspacesState
         const onActive = () => this.workspaceModel.active = true;
         const onRecent = () => this.workspaceModel.active = false;
         return <>
-        <Header title="Workspaces" subtitle="Open and Share Workspaces"/>
-        
-        <div className="lg:px-28 px-10 pt-4 flex">
-            <input type="text" placeholder="Search Workspace"  onChange={(v) => {console.log(v)}} />
-
-            <Toggle entries={[{
-                title: 'Active',
-                onActivate: onActive
-            }, {
-                title: 'Recent',
-                onActivate: onRecent
-            }]} />
-        </div>
-        <div className="lg:px-28 px-10 pt-4 flex flex-col">
-            {
-                this.state?.workspaces.map(e => {
-                    return <WorkspaceEntry key={e.workspace.id} {...e} />
-                })
-            }
-        </div></>;
+            <Header title="Workspaces" subtitle="Manage past workspaces" />
+
+            <div className="lg:px-28 px-10 pt-4 flex">
+                <div className="flex">
+                    <div className="py-4">
+                        <svg xmlns="http://www.w3.org/2000/svg" width="14" height="14" viewBox="0 0 490 490">
+                            <path fill="none" stroke="#777" stroke-width="36" stroke-linecap="round"
+                                d="m280,278a153,153 0 1,0-2,2l170,170m-91-117 110,110-26,26-110-110" />
+                        </svg>
+                    </div>
+                    <input className="border-0 text-sm text-gray-400" type="text" placeholder="Search Workspace" onChange={(v) => { console.log(v) }} />
+                </div>
+                <div className="flex-1" />
+                <div className="py-3">
+                    <DropDown entries={[{
+                        title: 'Status: Active',
+                        onClick: onActive
+                    }, {
+                        title: 'Status: Recent',
+                        onClick: onRecent
+                    }]} />
```

Comment:
As a user, I have no idea what these mean, and they're not mutually exclusive.

I'd still vote for:

- `Top Workspaces` (by default, only running or pinned)
- `All Workspaces` (if you didn't find what you wanted)